src/components/ui/Error.tsx

```diff
@@ -46,11 +46,11 @@ const Error = () => {
                 {cors &&
                 <div>
                     This dataset is hosted on a different website and your browser is blocking it for security reasons. 
-                    It's called a "CORS" issue (Cross-Origin Resource Sharing). 
+                    It&apos;s called a &quot;CORS&quot; issue (Cross-Origin Resource Sharing). 

```

Comment:
this should work better:

```
<>{"do this"} </>
```
using those weird scape symbols becomes cumbersome.